src/Console/Command/ListFixersCommand.php

```diff
@@ -0,0 +1,524 @@
+    /**
+     * @param string           $name
+     * @param RuleSetInterface $set
+     */
+    private function processRuleSet($name, RuleSetInterface $set)
+    {
+        /** @var bool $value */
+        foreach ($set->getRules() as $rule => $value) {
+            $this->fixerList[$rule]['in_set'][] = $name;
+            $this->fixerList[$rule]['is_inherited'] = true;
+        }
+    }
+
+    /**
+     * @param FixerInterface $fixer
+     */
+    private function processFixer(FixerInterface $fixer)
+    {
+        $this->fixerList[$fixer->getName()]['name'] = $fixer->getName();
+        $this->fixerList[$fixer->getName()]['is_configured'] = $this->isFixerConfigured($fixer);
+        $this->fixerList[$fixer->getName()]['is_enabled'] = $this->isFixerEnabled($fixer);
+        $this->fixerList[$fixer->getName()]['is_enabled_Through_inheritance'] = $this->isFixerEnabledThroughInheritance($fixer);
+        $this->fixerList[$fixer->getName()]['is_risky'] = $this->isFixerRisky($fixer);
+        $this->fixerList[$fixer->getName()]['is_inherited'] = false;
+        $this->fixerList[$fixer->getName()]['is_deprecated'] = $this->isFixerDeprecated($fixer);
+        $this->fixerList[$fixer->getName()]['is_custom'] = $this->isCustomFixer($fixer);
+    }
+
+    /**
+     * @param OutputInterface $output
+     *
+     * @return Table
+     */
+    private function buildTable(OutputInterface $output)
+    {
+        $table = new Table($output);
+
+        $rows = $this->filterFixers();
+        $table->setRows($rows);
+
+        $columns = [
+            sprintf('Fixer (%d)', \count($rows)),
+        ];
+
+        if (!$this->hideConfigured) {
+            $columns[] = sprintf('Configured (%d)', $this->countConfiguredFixers);
+        }
+
+        if (!$this->hideEnabled) {
+            $columns[] = sprintf('Enabled (%d)', $this->countEnabledFixers);
+        }
+
+        if (!$this->hideRisky) {
+            $columns[] = sprintf('Risky (%d)', $this->countRiskyFixers);
+        }
+
+        if (!$this->hideInherited) {
+            $columns[] = sprintf('Inherited (%d)', $this->countInheritedFixers);
+        }
+
+        if (!$this->hideDeprecated) {
+            $columns[] = sprintf('Deprecated (%d)', $this->countDeprecatedFixers);
+        }
+
+        if (!$this->hideCustom) {
+            $columns[] = sprintf('Custom (%d)', $this->countCustomFixers);
+        }
+
+        $table->setHeaders([$columns]);
+
+        return $table;
+    }
+
+    /**
+     * @return array
+     */
+    private function filterFixers()
+    {
+        $hideConfigured = $this->hideConfigured;
+        $hideEnabled = $this->hideEnabled;
+        $hideRisky = $this->hideRisky;
+        $hideInherited = $this->hideInherited;
+        $hideDeprecated = $this->hideDeprecated;
+        $hideCustom = $this->hideCustom;
+
+        $rows = array_filter($this->fixerList, function (array $fixer) use ($hideConfigured, $hideEnabled, $hideRisky, $hideInherited, $hideDeprecated, $hideCustom) {
```

Comment:
No need to introduce variables and pass them through `use` to the closure - `$this` can be used inside as long as closure is not defined as `static`.